### DISCUSSION TITLE
MIDI In Conversion

### DISCUSSION THREAD

**Discussion by Bananagab** [alf45tar/PedalinoMini#380] [Ideas]
Hi,

It's maybe a feature that already exists but I have no ESP32 on hand to test the software. Would it be possible to route the MIDI In from Bluetooth to a MIDI Output after "transposing" the midi cc number?

For example, I send CC #8 from a bluetooth MIDI controller and I want the Pedalino to convert that CC #8 to CC #1.

**Comment by alf45tar** [comment]
Midi routing is supported on any interface but there is no transcoding. What received from one interface is sent to other interface without any change.
